active/0000-devcert-ssl.md

```diff
@@ -0,0 +1,65 @@
+- Start Date: 2017-04-28
+- RFC PR: (leave this empty)
+
+# Summary
+
+Add support for on-the-fly SSL certificates in development via devcert.
+
+# Motivation
+
+Many of the newest web platform features (i.e. service workers) require HTTPS transport. Generating
```

Comment:
I thought that you could also test Service Workers against `localhost`:

>  In order to facilitate local development, localhost is considered a secure origin by browsers as well.

[MDN](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API/Using_Service_Workers)